packages/api/tests/lambdas/test-internal-reconciliation-report.js

```diff
@@ -100,12 +95,7 @@ test.afterEach.always(async (t) => {
     knexAdmin: t.context.knexAdmin,
     testDbName: t.context.testDbName,
   });
-  await Promise.all(
-    flatten([
-      t.context.bucketsToCleanup.map(recursivelyDeleteS3Bucket),
-      new models.ReconciliationReport().deleteTable(),
-    ])
-  );
+  await Promise.all(flatten(t.context.bucketsToCleanup.map(recursivelyDeleteS3Bucket)));
   await esClient.client.indices.delete({ index: esIndex });
```

Comment:
So I think this test is more focused on what is actually in the recon report created, not the DB record (which doesn't contain the report, just the location where the report was created). So since it's being written to the same DB as collections, granules, etc, that DB is already being created for the tests, and the Dynamo table is not needed. And the tests are not grabbing the DB recon report record, so there are no changes needed there. The tests instead are just examining/testing the innards of the report being created by `const report = await internalRecReportForGranules({ knex });`